usage_record.go

```diff
@@ -1,33 +1,26 @@
 package stripe
 
 const (
-	// UsageRecordParamsActionIncrement indicates that if two usage records conflict
-	// (i.E. are reported a the same timestamp), their Quantity will be summed
-	UsageRecordParamsActionIncrement string = "increment"
-
-	// UsageRecordParamsActionSet indicates that if two usage records conflict
-	// (i.E. are reported a the same timestamp), the Quantity of the most recent
-	// usage record will overwrite any other quantity.
-	UsageRecordParamsActionSet string = "set"
+	UsageRecordActionIncrement string = "increment"
+	UsageRecordActionSet       string = "set"
```

Comment:
Should these be a new type like `type UsageRecordAction string`?

Reply:
so the reason I did not put a type it's because it's only used as a parameter. Happy to add one though